### ISSUE TITLE
Layer height information not saved in AMF file

### ISSUE THREAD

**Issue by zsolta** [alexrj/Slic3r#2556]
First I would like to thank your hard work, Slic3r is a great piece of software. 

Change/override the layer height information for a given par on the plater using Settings... -> Layer and  save the file as an AMF file using Plater -> Export AMF ...

Delete all parts from Plater, then Add... the previously saved AMF file to it. If you check the layer overriding tab in the Settings you'll see that the values inputted earlier are missing, the layer table is empty.

On the other hand, if you change the Part Settings for a given part and export the file as an AMF, the information will be restored correctly.

In other words if you change the Part Settings and the Layers Height for a part, you save that part as an AMF then later you add that AMF file to the plate, only the Part Settings will be restored, the Layers height info is lost.

Not sure if this is a bug, a missing feature or a limitation of the AMF file format, but certainly it is a source of errors for complex slicing configurations (for me at least).

**Comment by alexrj** [comment]
Well, yes, the layer heigths set in the *Layers* tab of the *Settings* window are not saved to AMF. Only the Part settings are. Maybe yes, we should save those values as well.
